api/lock.go

```diff
@@ -123,7 +123,7 @@ func (c *Client) LockOpts(opts *LockOptions) (*Lock, error) {
 
 // Lock attempts to acquire the lock and blocks while doing so.
 // Providing a non-nil stopCh can be used to abort the lock attempt.
-// Returns a channel that is closed if our lock is lost or an error.
+// Returns a channel that is nil if aborted, closed if our lock is lost or an error.
```

Comment:
```suggestion
//If the lock is aborted, it returns a channel that is nil. If the lock is lost or an error occurred, it returns a channel the is close. 
```

Reply:
I think the last bit of this suggestion should read "it returns a channel that is closed."

Reply:
Sounds good. Updated.